src/main/java/net/sf/jabref/openoffice/CitationManager.java

```diff
@@ -140,20 +140,20 @@ public void showDialog() {
     static class CitEntry implements Comparable<CitEntry> {
 
         private final String refMarkName;
-        private String pageInfo;
+        private Optional<String> pageInfo;
         private final String context;
-        private final String origPageInfo;
+        private final Optional<String> origPageInfo;
 
 
-        public CitEntry(String refMarkName, String context, String pageInfo) {
+        public CitEntry(String refMarkName, String context, Optional<String> optional) {
```

Comment:
I have added two more constructor, one (String, String), one (String, String, String). However, those are only used for testing. The only place where a constructor is used in the code is the above one and then I do not think it makes sense to add a .orElse(null) to remove the Optional-constructor.